### DISCUSSION TITLE
trouble with autodiscovery patroni

### DISCUSSION THREAD

**Discussion by glushakov** [weaponry/pgscv#16] [General]
Hi.
I have an issue with discover patroni on RHEL.
Run pgscv with debug:
{"level":"debug","service":"pgscv","time":"2021-10-22T17:50:32+03:00","message":"auto-discovery: looking up for new services..."}
{"level":"debug","service":"pgscv","time":"2021-10-22T17:50:32+03:00","message":"auto-discovery [patroni]: analyzing process with pid 2608"}
{"level":"warn","service":"pgscv","time":"2021-10-22T17:50:32+03:00","message":"auto-discovery [python3]: discovery failed: patroni YAML configuration file not found; skip"}

[root]# ps ax | grep patroni
 842679 ?        Ssl    0:00 /usr/bin/python3.6 /usr/local/bin/patroni /etc/patroni/postgres.yml

[root]# ps ax | grep 2608
   2608 ?        Ssl   49:56 /usr/bin/python3 /usr/bin/goferd --foreground

So, autodiscover found first python process and decide that is patroni. Has any workaround to tell pgscv when he can found patroni yaml file?
Found in WIKI about PATRONI_URL env variable , but dont understand how to use it.
Thanks

**Comment by lesovsky** [comment]
Hi,
Create an issue about this and specify RHEL version.

About PATRONI_URL - Patroni provides [REST API](https://patroni.readthedocs.io/en/latest/rest_api.html), you can specify url to API in `PATRONI_URL` env.

**Comment by glushakov** [comment]
Ok, thx.
here https://github.com/weaponry/pgscv/issues/17
